tests/model/test_rnn.py

```diff
@@ -31,7 +32,9 @@ def rnn_types(draw) -> st.SearchStrategy[RNNType]:
     forget_gate_bias=st.one_of(
         st.none(), st.floats(min_value=-10.0, max_value=10.0, allow_nan=False)
     ),
+    batch_first=st.booleans(),
 )
+@settings(deadline=4000)
```

Comment:
See above comment about `deadline`.

Reply:
removed - this is fine when running tests on the cpu. I've just realised that the test suite is much slower on my gpu than cpu locally :sweat_smile: